gdk4-win32/src/win32_display.rs

```diff
@@ -24,4 +26,69 @@ impl Win32Display {
             }
         }
     }
+
+    #[doc(alias = "gdk_win32_display_add_filter")]
+    pub fn add_filter<F>(&self, filter_func: F) -> Win32DisplayFilterHandle<F>
```

Comment:
This will be impossible to store in a struct because of the type parameter. Can you remove it and simply store a `Box<dyn Fn ...>` instead?

Reply:
ok, lets try that, not sure it's correct. please check! :)